lib/utils/routes.js

```diff
@@ -30,6 +34,11 @@ function routes(app, middleware, options) {
     createReadStream(join(clientBasePath, 'live.html')).pipe(res);
   });
 
+  app.get('/invalidate', (_req, res) => {
```

Comment:
This should better be prefix with `webpack-dev-server` maybe `/webpack-dev-server/invalidate`. Otherwise `/invalidate` is unusable for the user and apps using `/invalidate` in their router break.